src/mem3/src/mem3_reshard_job.erl

```diff
@@ -50,6 +50,7 @@
 
 % Batch size for internal replication topoffs
 -define(INTERNAL_REP_BATCH_SIZE, 2000).
+-define(DEFAULT_REXI_TIMEOUT, 600000).
```

Comment:
To reduce the chance of a timeout, especially when resharding jobs run at a low ioq priority we could try to reduce the default batch size and use 500 like we use for the internal replicator in addition to the timeout option.